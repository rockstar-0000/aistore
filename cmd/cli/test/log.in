mkdir -p /tmp/aislogs
ais log get $RANDOM_TARGET /tmp/aislogs
ais log get $RANDOM_TARGET /tmp/aislogs --all
ais log get $RANDOM_TARGET /tmp/aislogs --severity e
ais cluster download-logs - // FAIL
ais cluster download-logs /tmp/aislogs // IGNORE
rm -rf /tmp/aislogs
