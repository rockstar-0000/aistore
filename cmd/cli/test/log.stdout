^Downloading.*$
Done
^Downloading.*tar.gz$
Done
^Downloading.*$
Done
